internal/cmd/appgroups/crtapp.go

```diff
@@ -69,5 +69,4 @@ func init() {
 
 	_ = CreateAppCmd.MarkFlagRequired("name")
 	_ = CreateAppCmd.MarkFlagRequired("app-name")
-	_ = CreateAppCmd.MarkFlagRequired("prods")
```

Comment:
@ascopes - are you looking at creating an appgroup app? This is for creating a developer app. From the code I see that apiproduct is not mandatory while creating an appgroup app